source/features/conversation-navigation.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import ChevronLeftIcon from 'octicon/chevron-left.svg';
+import ChevronRightIcon from 'octicon/chevron-right.svg';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import SearchQuery from '../github-helpers/search-query';
+import looseParseInt from '../helpers/loose-parse-int';
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+function getButton(direction: string): HTMLAnchorElement {
+	return (
+		<a
+			aria-disabled="true"
+			className="btn btn-lg BtnGroup-item"
+			aria-label={`Navigate to ${direction} Conversation`}
+			data-hotkey={direction.charAt(0)}
+		>
+			{direction === 'next' ? <ChevronRightIcon/> : <ChevronLeftIcon/>}
+		</a>
+	) as unknown as HTMLAnchorElement;
+}
+
+async function init() {
+	const previousButton = getButton('previous');
+	const nextButton = getButton('next');
+
+	select('.gh-header-meta')!.append(
+		<div className="BtnGroup ml-2">
+			{previousButton}
+			{nextButton}
+		</div>
+	);
+
+	const conversationNumber = getConversationNumber();
+	const {list, query, page} = await getConversationList(conversationNumber);
+	const conversation = list.find(item => item.number === conversationNumber);
```

Comment:
You pass `conversationNumber` to a function, why does that function return a list instead of the specific conversation you specifically requested? It should return `conversation` itself.

Reply:
Renamed `getConversationList` to `getConversationMeta` and moved all that logic for finding next/previous inside